### DISCUSSION TITLE
How do I increase the memory for Qodana CLI?

### DISCUSSION THREAD

**Discussion by sdyson** [JetBrains/Qodana#274] [Q&A] [answered]
I'm running Qodana CLI in native mode in an Azure Pipeline. The scan is taking a long time and I'm getting warnings about memory: 

##[warning]Free memory is lower than 5%; Currently used: 95.80%

How can I allocate more memory to the Java process?

**Comment by scottdfedorov** [comment, marked as answer]
https://www.jetbrains.com/help/qodana/docker-image-configuration.html#docker-config-reference-docker-environment-heap-size
Pass the -e flag to the args param: https://www.jetbrains.com/help/qodana/qodana-azure-pipelines.html#Configuration
